bothub/common/views.py

```diff
@@ -1,18 +1,14 @@
 from django.shortcuts import get_object_or_404
-from django.http import HttpResponse
+from django.http import HttpResponseRedirect
 from django.core.exceptions import ValidationError
 from django.contrib.admin.views.decorators import staff_member_required
 from .models import RepositoryUpdate
 
 
 @staff_member_required
 def download_bot_data(self, update_id):  # pragma: no cover
-    update = get_object_or_404(RepositoryUpdate, id=update_id)
+    update = get_object_or_404(RepositoryUpdate, pk=update_id)
     if not update.trained_at:
         raise ValidationError('Update #{} not trained at.'.format(update.id))
```

Comment:
Recommended to use f-strings to ensure a code more readable. Use of *pk* instead of *id*:
`raise ValidationError(f'Update #{update.pk} not trained at.')